docs/refactoring/react_flow_workflow_canvas_plan.md

```
@@ -391,7 +412,32 @@ const handleStartPhase = async (phaseId: string) => {
+- Status includes: `overallPass`, `totalChecks`, `passedChecks`, `failedChecks`
+
+**Node Display:**
+- StepNode and LoopNode display pre-validation status when available
+- Green badge (emerald) with ShieldCheck icon when passed
+- Red badge (red) with ShieldX icon when failed
+- Shows check counts: "X/Y checks passed" or "X/Y checks passed (Z failed)"
+- Appears in node content area, below success criteria
+- Only shown after pre-validation completes (real-time updates from events)
+
+**Event Flow:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)